package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/rpx-xui-common-lib",
-  "version": "0.0.24",
+  "version": "0.0.25",
```

Comment:
version should not just be updated in this package json, but also in the package json in the library directory.